frontend/src/metabase/components/EmptyState.jsx

```diff
@@ -29,7 +29,18 @@ const LegacyImage = props =>
       className={props.imageClassName}
     />
   ) : null;
-
+/**
```

Comment:
Add type to stop typescript from complaining as `EmptyState` will be used in a TypeScript file.

Reply:
Yep, using JSDoc here is good 👍 . I think there are a few other places where we've used it to improve type inference without moving an entire file to typescript.